modules/ref-dashboard-configuration-options.adoc

```diff
@@ -21,6 +36,14 @@ disableCustomServingRuntimes` | `false` | Shows the *Settings → Serving runtim
 | `dashboardConfig:
 disableDistributedWorkloads` | `false` | Shows the *Distributed workloads* option in the dashboard navigation menu. To hide this menu option, set the value to `true`.
 | `dashboardConfig:
+disableHardwareProfiles` (Technology Preview) | `true` | Hides the *Settings → Hardware profiles* option in the dashboard navigation menu, and shows the *Settings → Accelerator profiles* option if that option is enabled. 
+
```

Comment:
suggest ending change to:
...if the 'disableAcceleratorProfiles'  is set to 'false'.

Reply:
Updated, thank you!